### ISSUE TITLE
Candy Shop/ sweet shop idea

### ISSUE THREAD

**Issue by DragonBallBreakersRandomizer** [SmashMC-Development/Suggestions#778]
**Describe the suggestion**
At warp mart or its own location, maybe make a shop that sells candy-related (sweet shop) items such as: Rare candy (regular one u sell them for 2k each at one of the marts), (exp candy XS, s, m), Dynamax Candy, Rage Candy Bar, Sweet Apple, (all the sweets for alcreme. ie ribbon, berry,) Clover, Flower, Love, Star, Strawberry), and sweetheart candy
**Which servers does this apply to?**
ALL

**Additional context**
maybe at warp mart

**Comment by AngryRedHead101** [comment]
Maybe also add a herb shop as well, selling items like power herbs, revival herbs, mental herb and white herb, Power and revival herb could be sold for 10k each while mental and white could be 5k, keeps prices fair and affordable for all, seeing as how rare it is to find power herbs anymore

**Comment by DragonBallBreakersRandomizer** [comment]
Yeah a herb shop could be nice or maybe sell herbs with my flower shop idea #777
